fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ImportJobListener.java

```diff
@@ -8,29 +8,46 @@
 
 import java.util.HashMap;
 import java.util.List;
+import java.util.logging.Logger;
 
 import javax.batch.api.listener.JobListener;
+import javax.batch.operations.JobOperator;
+import javax.batch.runtime.BatchRuntime;
+import javax.batch.runtime.JobExecution;
 import javax.batch.runtime.context.JobContext;
 import javax.inject.Inject;
 
 public class ImportJobListener implements JobListener {
+    private static final Logger logger = Logger.getLogger(ImportJobListener.class.getName());
     @Inject
     JobContext jobContext;
 
-    private long jobStartTimeInMS, jobEndTimeInMS;
-
     public ImportJobListener() {
 
     }
 
-    @SuppressWarnings("unchecked")
+
+    @SuppressWarnings({"unchecked" })
```

Comment:
which warning is being supressed?  can we avoid it?

Reply:
List<ImportCheckPointData> partitionSummaries = (List<ImportCheckPointData>)jobContext.getTransientUserData();

Reply:
List<ImportCheckPointData> partitionSummaries = (List<ImportCheckPointData>)jobContext.getTransientUserData();

Reply:
"List<ImportCheckPointData> partitionSummaries = (List<ImportCheckPointData>)jobContext.getTransientUserData(); "

Reply:
`List<ImportCheckPointData> partitionSummaries = (List<ImportCheckPointData>)jobContext.getTransientUserData();`

Reply:
can we apply the annotation to this field instead of to the whole method then?

Reply:
yes, we can. :)

Reply:
done